matrix-meetings-bot/src/service/MeetingService.ts

```diff
@@ -363,15 +365,25 @@ export class MeetingService {
     const oldMeeting = room.meeting;
     const newMeeting: IMeeting = { ...room.meeting };
 
-    newMeeting.calendar = meetingDetails.calendar ?? newMeeting.calendar;
+    // change data model if meeting is OX with non-empty rrules
+    const meetingTime = migrateMeetingTime(
```

Comment:
See my comment in `MeetingClient.tsx`.

Reply:
changed to be like in the `MeetingClient`